project-resources/INFRASTRUCTURE_MENU.md

```diff
@@ -25,7 +25,7 @@ The OpenJS Foundation will purchase 2-year wildcard SSL certs for each project's
 By default all OpenJS Foundation projects have open source repositories in their own GitHub Organizations.  The OpenJS Foundation admin account must be added as administrator for each repository.  Two-factor authentication must be required for everyone in the organization.
 
 ## Build (CI/CD)
-Projects which require CI/CD should attempt to use solutions which are free to open source projects, such as https://travis-ci.org/.  Non-free options may be requested, subject to budget approval.
+Projects which require CI/CD should attempt to use solutions which are free to open source projects.  Non-free options may be requested, subject to budget approval.
```

Comment:
alternative:
```suggestion
Projects which require CI/CD should attempt to use solutions which are free to open source projects, such as Github Actions. Non-free options may be requested, subject to budget approval.
```

Reply:
I'm OK either way.

Reply:
I'd be okay with this but also medium -1 against us including free tools that projects can get themselves in the Foundation's "Infrastructure Menu". Feels wildly underwhelming and useless.